CI/appveyor.after_success.ps1

```diff
@@ -8,11 +8,12 @@ windeployqt.exe --release mudlet.exe
 
 Remove-Item * -include *.cpp, *.o
 
-if ("$Env:APPVEYOR_REPO_TAG" -eq "false") {
-  $DEPLOY_URL = "https://ci.appveyor.com/api/buildjobs/$Env:APPVEYOR_JOB_ID/artifacts/src%2Fmudlet.zip"
-} else {
-  git clone https://github.com/Mudlet/installers.git C:\projects\installers
+#if ("$Env:APPVEYOR_REPO_TAG" -eq "false") {
+#  $DEPLOY_URL = "https://ci.appveyor.com/api/buildjobs/$Env:APPVEYOR_JOB_ID/artifacts/src%2Fmudlet.zip"
+#} else {
+  git clone https://github.com/keneanung/mudlet-installers.git C:\projects\installers
   cd C:\projects\installers\windows
+  git checkout AddWindowsCodeCertificate
```

Comment:
This seems a little weird. Why it is necessary to check out a file that should already be present? :confused:

Reply:
That's to check out the correct branch of the `installers` repo.